models/dvmdostem/R/model2netcdf.dvmdostem.R

```diff
@@ -13,7 +13,7 @@ library(lubridate)
 ##' @title Write data into PEcAn shaped output file.
 ##' @param y_starts a list of years, i.e.: 1901, 1902, 1903, etc.
 ##' @param outdir a path to the location where were we will look for dvmdostem outputs and write PEcAn outputs.
-##' @param pecan_requested_vars space separated string listing the variables to process (PEcAn names).
+##' @param pecan_requested_vars comma separated string listing the variables to process (PEcAn names).
```

Comment:
@tobeycarman did you rebuild the docs after making the changes to the documentation?

Reply:
I had not done that, but I think I finally got everything comitted and I think the build is passing now.